canvas_modules/common-canvas/src/common-canvas/common-canvas.scss

```diff
@@ -112,6 +116,21 @@ $bottom-panel-border-color: $ui-03;
 	height: 100%;
 }
 
+.top-panel {
+	display: flex;
+	flex-direction: column;
+	position: absolute;
+	left: 0;
+	right: 0;
+	top: $toolbar-button-height;
+	border-top: $panel-border-width solid $panel-border-color;
```

Comment:
Should this be border-bottom instead?

Reply:
Yes, great catch -- thank you! I fixed it.